.travis.yml

```diff
@@ -1,23 +1,15 @@
+version: ~> 1.0
+import: scala/scala-dev:travis/default.yml
+
 language: scala
-jdk:
-  - openjdk8
-  - openjdk11
+env:
+  - ADOPTOPENJDK=8
+  - ADOPTOPENJDK=11
 scala:
   - 2.12.12
-  - 0.27.0-RC1
   - 2.13.3
+  - 3.0.0-M1
```

Comment:
I think this will make scala 3.0.0-M1 the default when starting sbt because sbt-travis-ci picks the last in the list as the `scalaVersion`. I think 2.13.3 should stay the default.